website/util/time.py

```diff
@@ -10,7 +11,25 @@ def get_timestamp():
 def throttle_period_expired(timestamp, throttle):
     if not timestamp:
         return True
-    elif isinstance(timestamp, datetime.datetime):
-        return (datetime.datetime.utcnow() - timestamp).total_seconds() > throttle
+    elif isinstance(timestamp, datetime):
+        return (datetime.utcnow() - timestamp).total_seconds() > throttle
     else:
         return (get_timestamp() - timestamp) > throttle
+
+
+def generate_expiration_time(duration, base=datetime.utcnow(), utc=False):
```

Comment:
Yes, this should be removed. It was used for cookie flag `Max-Age` and `Expired` which is now removed.